beeline/src/java/org/apache/hive/beeline/TableOutputFormat.java

```diff
@@ -122,18 +122,10 @@ ColorBuffer getOutputString(Rows rows, Rows.Row row, String delim) {
 
       if (row.isMeta) {
         v = beeLine.getColorBuffer().center(row.values[i], row.sizes[i]);
-        if (beeLine.getOpts().getColor() && rows.isPrimaryKeyCol(i)) {
-          buf.cyan(v.getMono());
```

Comment:
Retrieving the primary key in Beeline is just for coloring the output.  Removing this code blocks is safe.